<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBaseClass</class>
 <widget class="QMainWindow" name="DataBaseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSnap">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Snapshots</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDischarge">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Discharge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>871</width>
        <height>468</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DataBase.qrc"/>
 </resources>
 <connections/>
</ui>
